app/components/Views/confirmations/SendFlow/Confirm/index.js

```diff
@@ -279,10 +285,28 @@ class Confirm extends PureComponent {
     setProposedNonce(proposedNonce);
   };
 
-  getAnalyticsParams = () => {
+  getAnalyticsParams = (transactionMeta) => {
     try {
-      const { selectedAsset, gasEstimateType, chainId } = this.props;
+      const { selectedAsset, gasEstimateType, chainId, isSmartTransaction } =
+        this.props;
       const { gasSelected, fromSelectedAddress } = this.state;
+      const { SmartTransactionsController } = Engine.context;
+
+      let smartTransactionMetadata = {};
+      if (transactionMeta) {
+        const smartTransaction =
+          SmartTransactionsController.getSmartTransactionByMinedTxHash(
+            transactionMeta.hash,
+          );
+
+        if (smartTransaction) {
+          smartTransactionMetadata = {
+            duplicated: smartTransaction.statusMetadata.duplicated,
+            timedOut: smartTransaction.statusMetadata.timedOut,
+            proxied: smartTransaction.statusMetadata.proxied,
+          };
+        }
+      }
```

Comment:
Can be simplified and remove `let`.

```suggestion
      const smartTransaction = transactionMeta
        ? SmartTransactionsController.getSmartTransactionByMinedTxHash(
            transactionMeta.hash,
          )
        : null;
      const smartTransactionMetadata = smartTransaction
        ? {
            duplicated: smartTransaction.statusMetadata.duplicated,
            timedOut: smartTransaction.statusMetadata.timedOut,
            proxied: smartTransaction.statusMetadata.proxied,
        } : {};
```